2021/opmodes/UltimateGoalTeleop.java

```diff
@@ -156,6 +156,7 @@ public void handleEvent(RobotEvent e) {
                         //activate launching mech
                         launchMechLeft.setPower(0.5);
                         launchMechRight.setPower(-0.15);
+                        break;
```

Comment:
This is why, when I'm demonstrating software and am filling out a switch statement's case labels, the breaks always, always, always go in before any of the software.  Very, very common source of error.  Inadvertent fall-through.